test/test-spawn.c

```diff
@@ -1861,7 +1861,9 @@ int spawn_stdin_stdout(void) {
       c = c - w;
     }
   }
+#ifndef __SUNPRO_C
   return 2;
+#endif
```

Comment:
This function should probably just be changed to return `void`, with the call site in run-tests.c changed from:
```c
return spawn_stdin_stdout();
```
To:
```c
spawn_stdin_stdout();
return 1;
```